servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/GlobalExecutionContext.java

```diff
@@ -152,7 +152,8 @@ public Cancellable schedule(final Runnable task, final long delay, final TimeUni
     }
 
     private static void log(final Logger logger, final String name, final String methodName) {
-        logger.debug("Closure of \"{}\" was initiated using {} method. Closing the global instance before " +
-                "closing all resources that use it may result in unexpected behavior.", name, methodName);
+        logger.info("Closure of \"{}\" was initiated using {} method. Closing the global instance before " +
+                "closing all resources that use it may result in unexpected behavior.", name, methodName,
+                new Exception("Stack trace"));
```

Comment:
I have always used `new Throwable` for stack dumps to make it clear that it is not actually an exception.